card/mod/01_core/set/all/fetch.rb

```diff
@@ -86,10 +105,15 @@ def known? mark
     card.present?
   end
 
-  def expire name
+  def expire name, subcards=false
     #note: calling instance method breaks on dirty names
     key = name.to_name.key
     if card = Card.cache.read( key )
+      if subcards
+        card.expire_subcards
+      else
+        card.preserve_subcards
+      end
```

Comment:
the workaround for the "expire" problem. There are many cases where the subcards have to survive a "expire" call